From 36bd76145a20479c867c7868e5c20396b2b9e30a Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Tue, 27 May 2014 16:58:42 +0200
Subject: [PATCH] setup-apkrepos: force install virtual package

it is tmpfs and we know it will be removed on reboot. Infact, we want
it to be removed on reboot.
---
 setup-apkrepos.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup-apkrepos.in b/setup-apkrepos.in
index ad9c309..e84b928 100644
--- a/setup-apkrepos.in
+++ b/setup-apkrepos.in
@@ -161,7 +161,7 @@ done
 # install alpine-mirrors if its not already there
 to_uninstall=
 if ! apk info --quiet --installed alpine-mirrors; then
-	apk add --quiet --virtual .setup-apkrepos alpine-mirrors
+	apk add --force --quiet --virtual .setup-apkrepos alpine-mirrors
 	to_uninstall=.setup-apkrepos
 fi
 
-- 
1.9.3

